pennylane/tape/qubit_param_shift.py

```diff
@@ -175,7 +176,7 @@ def processing_fn(results):
                 array[float]: 1-dimensional array of length determined by the tape output
                 measurement statistics
             """
-            results = np.squeeze(results)
+            results = np.squeeze([toarray(res) for res in results])

```

Comment:
🙌

Reply:
I wonder if it is worth putting a try-except around this?

Reply:
(@cvjjm - the tests you added to ensure object results are returned still passes, but wondering if this line will have any implications wrt your current workflow).

Reply:
Thanks for thinking of me!  I don't fully understand what made this change necessary, but from a quick test I could not see any negative impact.